data/wiki/hello-world.md

```diff
@@ -247,54 +199,46 @@ module Hello (Time : Mirage_time.S) = struct
 end
 ```
 
-To veteran OCaml programmers among you, this might look a little odd: We have a
-main `Hello` module parameterised by a module (`Time`, of type `Mirage_time.S`) that contains a method `start` taking an ignored parameter `_time` (an instance of a `time`).  This is the basic structure required to make this a MirageOS unikernel
-rather than a standard OCaml POSIX application.
+We define a main `Hello` module parameterised by a module `Time`, of type
+`Mirage_time.S`. The `Time` module provides the functionality enabling us to
+interact with the environment clock. Our `start` function also takes a parameter
+`_time` (an instance of a `time` which is ignored). This parameterization of a
+unikernel's main module and its `start` function is the basic structure required
+to make a MirageOS unikernel that can be built to run on any supported
+environment, rather than a standard OCaml POSIX application.
 
 The module type for our `Time` module, `Mirage_time.S`, is defined in an
-external package [mirage-time](https://github.com/mirage/mirage-time).  The name `S` for "the module type of things like this" is a common OCaml convention (comparable to naming the most-used type in a module `t`).  There are many packages defining module types for use in Mirage.
+external package [mirage-time](https://github.com/mirage/mirage-time).  The name
+pattern `Foo.S` for "the **s**ignature of `Foo` modules" is a common OCaml
+convention (comparable to naming the most-used type in a module `t`).  There are
+many packages defining module types for use in Mirage.
 
-The concrete implementation of `Time` will be supplied at
-compile time, depending on the target that you are compiling for. This
-configuration is stored in `config.ml`, so let's take a look:
+The concrete implementation of `Time` will be supplied at compile time,
+depending on the target that you are compiling for. This calls for some
+additional configuration  in `config.ml`, so let's take a look:
 
 ```ocaml
 $ cat tutorial/hello/config.ml
 
 open Mirage
 
 let main =
```

Comment:
It's a bit confusing that we're using `main` twice here. Maybe `unikernel` or `hello` would be clearer.